driver/razerkbd_driver.c

```diff
@@ -352,9 +357,13 @@ static void razer_set_device_mode(struct usb_device *usb_dev, unsigned char mode
     case USB_DEVICE_ID_RAZER_BLACKWIDOW_V3_MINI:
         report.transaction_id.id = 0x1F;
         break;
+    case USB_DEVICE_ID_RAZER_BLACKWIDOW_V3_PRO_WIRELESS:
     case USB_DEVICE_ID_RAZER_BLACKWIDOW_V3_MINI_WIRELESS:
         report.transaction_id.id = 0x9F;
         break;
+    default:
+        report.transaction_id.id = 0x1f;
+        break;
```

Comment:
Why is this block added?

Reply:
removed it for now, still keeping this "conversation" open, maybe it was super important for something